thirdparty/angle/include/CL/cl.h

```diff
@@ -0,0 +1,1929 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2020 The Khronos Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+#ifndef __OPENCL_CL_H
+#define __OPENCL_CL_H
+
+#include <CL/cl_version.h>
+#include <CL/cl_platform.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
I don't think we're using OpenCL at this time, so we can likely remove the `CL/` folder.

Reply:
Actually, I think we only ever include `EGL/` stuff, so we can remove everything but `EGL/` and I guess `KHR/`:
```
$ pwd
/home/akien/Projects/godot/godot.git/thirdparty/angle/include/EGL
[akien@cauldron EGL (angle++)]$ gdrg include
egl.h
20:#include <EGL/eglplatform.h>
32: * Default extensions included: None
33: * Additional extensions included: _nomatch_^
41:#include <KHR/khrplatform.h>
42:#include <EGL/eglplatform.h>

eglext.h
20:#include <EGL/eglplatform.h>
28: * Default extensions included: egl
29: * Additional extensions included: _nomatch_^
1480:#include "eglext_angle.h"

eglext_angle.h
7://   Currently we don't include this file directly, we patch eglext.h
8://   to include it implicitly so it is visible throughout our code.

eglplatform.h
13: * they can be included in future versions of this file.  Please submit changes
18:#include <KHR/khrplatform.h>
61:#include <windows.h>
69:#include <inspectable.h>
115:#include <X11/Xlib.h>
116:#include <X11/Xutil.h>
136:#include <kernel/image.h>
```

Though I'm not sure the current `KHR/` includes, if they're called, are properly sourced from `thirdparty/angle/includes/KHR/`. It might be using `thirdparty/glad/KHR/`.